01_easy_algorithms/05_count_lone_ones/05_count_lone_ones.rb

```diff
@@ -0,0 +1,17 @@
+def count_lone_ones(x)
+  arr1 = []
+  arr_x = x.digits.reverse
+  z = arr_x.length
+  idx=0
+  while idx < z
```

Comment:
```suggestion
  while idx < arr_x.length
```

since you're only using z in one place in this file, there's no need to assign that variable. Just use it directly here.